.eslintcache

```diff
@@ -1 +1 @@
"61","messages":"62","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"44"},{"filePath":"63","messages":"64","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"44"},{"filePath":"65","messages":"66","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"44"},{"filePath":"67","messages":"68","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"69","messages":"70","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"44"},{"filePath":"71","messages":"72","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"44"},{"filePath":"73","messages":"74","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"44"},{"filePath":"75","messages":"76","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"44"},{"filePath":"77","messages":"78","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"44"},{"filePath":"79","messages":"80","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"44"},{"filePath":"81","messages":"82","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"44"},"/Users/julianikolova/home-wall/src/index.js",[],["83","84"],"/Users/julianikolova/home-wall/src/js/App.js",[],"/Users/julianikolova/home-wall/src/js/auth/Scopes.js",[],"/Users/julianikolova/home-wall/src/js/containers/HomeWallsContainer.js",[],"/Users/julianikolova/home-wall/src/js/context/UserContext.js",[],"/Users/julianikolova/home-wall/src/js/api/HomeWallApi.js",[],"/Users/julianikolova/home-wall/src/js/components/HomeWalls.js",[],"/Users/julianikolova/home-wall/src/js/components/HomeWall.js",[],"/Users/julianikolova/home-wall/src/js/components/nav/NavBar.js",[],"/Users/julianikolova/home-wall/src/js/containers/ProblemEditorContainer.js",[],"/Users/julianikolova/home-wall/src/js/components/nav/MenuDrawer.js",[],"/Users/julianikolova/home-wall/src/js/components/views/ProblemView.js",[],"/Users/julianikolova/home-wall/src/js/components/views/ProblemList.js",[],"/Users/julianikolova/home-wall/src/js/components/login/Login.js",[],"/Users/julianikolova/home-wall/src/js/components/views/ProblemEditor.js",[],"/Users/julianikolova/home-wall/src/js/utils/RayCasting.js",[],"/Users/julianikolova/home-wall/src/js/utils/Holds.js",[],"/Users/julianikolova/home-wall/src/js/utils/Grades.js",[],"/Users/julianikolova/home-wall/src/js/components/Board.js",[],"/Users/julianikolova/home-wall/src/js/components/buttons/AddProblemButton.js",[],{"ruleId":"85","replacedBy":"86"},{"ruleId":"87","replacedBy":"88"},"no-native-reassign",["89"],"no-negated-in-lhs",["90"],"no-global-assign","no-unsafe-negation"]
```

Comment:
What is this and why is it generated?

Reply:
It for eslint to check style of code is correct. Shouldnt be checked in, ill remove and to .gitignore file later